src/components/WorkSpace/SignInAlert.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import Button from 'react-bootstrap/button'
```

Comment:
Can you add this dependency in `package.json` by running `npm install --save react-bootstrap/button` and commit the changes to `package.json`.

Reply:
alternately, I think you can 
```js
import { Button } from 'react-bootstrap'
```
because we already have `react-bootstrap` as a dependency.

Reply:
@asooge react-bootstrap docs recommend importing individual components as opposed to entire library. I also don't believe we'll need to add a dependency on each individual bootstrap component either... let me know if you find a source that says otherwise.

Reply:
Yes. The `{ }` syntax will import a single module rather than the entire library. 

Either way I'm trying to deploy to netlify but getting this error:
```
5:41:22 PM: ./src/components/WorkSpace/SignInAlert.js
5:41:22 PM: Cannot find module: 'react-bootstrap/button'. Make sure this package is installed.
5:41:22 PM: You can install this package by running: npm install react-bootstrap/button.
```

Reply:
FYI here is an example of netlify deployed site for `development` I would like to also deploy a version from your branch to test on mobile. https://stoic-bose-478137.netlify.app/#/

Reply:
Ahh, I see the problem now. It should be `react-bootstrap/Button` not lowercase button. Just pushed up the change. Apologies! This should hopefully resolve the deployment issue.

Reply:
Looks beautiful. I Haven't tested yet on mobile https://zen-rosalind-1d324b.netlify.app/#/

Reply:
I just checked it out real quick since I’m on my phone. The swipe down is a weeee bit too sensitive right now. (Scrolling down turns into swiping at the moment) I’ll have to take a look and fix the targeted area for swiping. Will have an update tomorrow!

Reply:
sounds good. I think it will automatically redeploy (same link) when you push to github.